code/game/objects/items/teleportation.dm

```diff
@@ -183,10 +183,7 @@
 		if(!computer.power_station || !computer.power_station.teleporter_hub)
 			continue
 
-		if((computer.power_station.machine_stat & (NOPOWER|BROKEN|MAINT)) || computer.power_station.panel_open)
-			continue
-
-		if((computer.power_station.teleporter_hub.machine_stat & (NOPOWER|BROKEN|MAINT)) || computer.power_station.teleporter_hub.panel_open)
```

Comment:
This is not a fix, ``is_operational`` explicitly checks for these values - you're doing exactly the same BUT removing the panel open code.

Reply:
Looks like this got accidentally merged in:

- #88566

Reply:
The initial commit in https://github.com/tgstation/tgstation/pull/88565/commits/73b457f1d0feef9a2585b8d2335259ce904bf79b got applied to both branches